pandas/io/json/json.py

```diff
@@ -226,7 +226,7 @@ def _write(self, obj, orient, double_precision, ensure_ascii,
         return serialized
 
 
-def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
+def read_json(path_or_buf=None, orient=None, typ='frame', dtype=None,
```

Comment:
@albertvillanova citing our conversation on the whatsnew - doesn't this signature change affect the other orients? Thought this method was shared across, hence why I was thinking this was an API change

Reply:
@WillAyd if you look at the implementation of `read_json`, you can check that `dtype` is only used for instantiating JsonReader; JsonReader only uses `dtype` for instantiating either FrameParser or SeriesParser (see method _get_object_parser); and both FrameParser and SeriesParser inherit from Parser, whose constructor sets `self.dtype=True` if `orient != 'table' and dtype is None`, thus recovering the behavior before the PR (default `dtype=True`) for all orients other than 'table'.

Reply:
Thanks for the explanation. I get your point but find it rather confusing / fragile from a code perspective to have that `None` handling so far removed from the actual user-facing method, hence my hesitation here

Reply:
Maybe, as @jreback pointed out, it would be more clear to set the default values (False for orient=table, and True for all other orients) in `read_json` table...